assets/src/edit-story/elements/media/getMediaSizePositionProps.js

```diff
@@ -34,8 +39,13 @@ function getMediaSizePositionProps(
   focalY
 ) {
   const ratio = width / height;
-  const oRatio =
+  let oRatio =
```

Comment:
Ditto: this code is propagating too far.

Reply:
Can you explain that comment, the above is unclear.

Reply:
I'll push an update. My point was that this all looks like can be done with less code. And hide the video/poster nuances within the media library itself.